package/firmware/ipq-wifi/Makefile

```diff
@@ -48,7 +48,8 @@ ALLWIFIBOARDS:= \
 	zte_mf289f \
 	zte_mf287 \
 	zte_mf287plus \
-	zyxel_nbg7815
+	zyxel_nbg7815 \
+	zyxel_wsq50
```

Comment:
Is the BDF for this board already included in the https://github.com/openwrt/firmware_qca-wireless repo?

Reply:
Hi robimarko, no it is not and I don't know how to rectify that

Reply:
Make a PR with the correct BDF-s wrapped by https://github.com/qca/qca-swiss-army-knife/blob/master/tools/scripts/ath10k/ath10k-bdencoder

Reply:
Thank you for assisting on IRC with this Robert, PR raised here; https://github.com/openwrt/firmware_qca-wireless/pull/21

Reply:
This is now merged in